web/src/layouts/MainLayout/MainLayout.js

```diff
@@ -16,7 +17,9 @@ const MainLayout = ({ children }) => {
             <li>
               <Link to={routes.home()}>
                 <Tooltip title="We need a logo!" >
-                  <div className="h-10 w-10 bg-indigo-500 rounded-full" data-tip="hello world" data-place="bottom"></div>
+
+                <img src={logo} style={{marginLeft : '50px'}}/>            
```

Comment:
Could you also remove the style property and change it to `className="ml-12"`, it should do the lame thing.